generators/csharp/codegen/src/ast/core/AstNode.ts

```diff
@@ -17,7 +18,9 @@ export abstract class AstNode {
         allNamespaceSegments: Set<string>,
         allTypeClassReferences: Map<string, Set<Namespace>>,
         rootNamespace: string,
-        customConfig: BaseCsharpCustomConfigSchema
+        customConfig: BaseCsharpCustomConfigSchema,
+        format = false,
```

Comment:
We might want to update the `toString` function to using an in-lined request too - I think the only reference is in `CSharpFile.ts`, and this will let us use optional parameters.